packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -192,6 +192,7 @@ - (void)testTransformFix {
   __block NSDictionary<NSString *, id> *initializationEvent;
   [player onListenWithArguments:nil
                       eventSink:^(id event) {
+                        NSLog(@"event: %@", event);
```

Comment:
This is the same logging that the assertion failure in the next line already does (which is less detail than what is in the `[event isKindOfClass:[FlutterError class]]` branch below), so you're not going to get new information here.

You could try logging information about the original error itself, in the `case AVPlayerItemStatusFailed:` section of FLTVideoPlayerPlugin.m, but I didn't have any luck with that in the CI.

Reply:
What if it's not a `FlutterError`? This logging helped me locally before to find out about the initialization. Let's see whaat it will print. 🤷